debian/control

```diff
@@ -1,29 +1,38 @@
 Source: wiringpi
-Section: electronics
 Priority: optional
-Maintainer: Ian Jackson <ijackson@chiark.greenend.org.uk>
-Standards-Version: 3.8.0
-Homepage: https://github.com/WiringPi/WiringPi
-Build-Depends: debhelper (>= 8)
-
-Package: libwiringpi2
+Maintainer: Rafael Diniz <rafael@rhizomatica.org>
```

Comment:
The WiringPi Community <wiringpi@gc2.at>

Reply:
Done.

Reply:
Btw, I did not see any link to subscribe or apply to be part of the community.